src/main/kotlin/com/ampnet/projectservice/persistence/repository/OrganizationRepository.kt

```diff
@@ -19,8 +19,15 @@ interface OrganizationRepository : JpaRepository<Organization, UUID> {
 
     fun findByNameAndCoop(name: String, coop: String): Optional<Organization>
 
-    fun findByNameContainingIgnoreCaseAndCoop(name: String, coop: String, pageable: Pageable): Page<Organization>
+    @Query(
+        "SELECT org FROM Organization org WHERE lower(org.name) like lower(concat('%', ?1,'%')) " +
+            "AND org.coop = ?2 AND org.active = true"
+    )
+    fun findAllActiveByNameAndCoop(name: String, coop: String, pageable: Pageable): Page<Organization>
 
     @Query("SELECT org FROM Organization org LEFT JOIN FETCH org.memberships WHERE org.uuid = :organizationUuid")
     fun findByIdWithMemberships(organizationUuid: UUID): Optional<Organization>
+
+    @Query("SELECT org FROM Organization org WHERE org.active = true")
+    fun findAllActive(pageable: Pageable): Page<Organization>
```

Comment:
Did you try JPA query `findByActive` and set the value true in organization service?